tsconfig.addons.json

```diff
@@ -8,12 +8,13 @@
     "checkJs": true,
     "declaration": true,
     "emitDeclarationOnly": false,
-    "outDir": "dist/addons",
+    "outDir": "dist/",
```

Comment:
You should not change this line. `dist/addons` is the root directory designed for addons.